Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
-
+# Byebug requires ruby >= 2.0.0 
+ruby '2.0.0'
```

Comment:
Yes, I agree. In that case, it would just be best to disregard this and accept Jeff's pull request, since his already specifies the latest ruby version and has all the additional awesome code for the test_helpers etc.